schemas/workflow_runs/definitions.yaml

```
@@ -21,6 +21,10 @@ workflow_run_shared:
         type: string
         minLength: 1
         maxLength: 128
+    sdk_token:
+      type: string
+      nullable: true
+      description: client token to use when loading this workflow run in the Onfido SDK.
```

Just for consistence:
```suggestion
      description: Client token to use when loading this workflow run in the Onfido SDK.
```